### PR TITLE
doc(xquery.xml): add in-built function map options

### PR DESCRIPTION
**Pull request by marmoure**
Documentation for https://github.com/eXist-db/exist/pull/3990

### CODE REVIEW

**Review by duncdrum — APPROVED** [review]
Changes are good, and valid, the errors reported by cypress are unrelated since currently `:latest` is broken 

see https://github.com/eXist-db/dashboard/issues/199

**Review by adamretter — APPROVED** [review]
LGTM :-)
Thanks Younes.

File: src/main/xar-resources/data/xquery/xquery.xml [hunk 1]
```diff
@@ -729,6 +729,65 @@
...
+        <sect2 xml:id="fn-serialize">
+            <title>Using <code>fn:serialize()</code></title>
+            <para>Another Serialization method that can be used is in-built function <literal>fn:serialize()</literal>.</para>
+            <para>
+                You can apply eXist-db legacy serialization parameters as well as W3C serialization parameters via a map.
+            </para>
+            <programlisting language="xquery" xlink:href="listings/listing-40.txt"/>
+            <note>
+                <para>
+                    For eXist-db legacy serialization parameters The option's QName must be exist:serialize as show in the example above
```
Comment (duncdrum) [inline src/main/xar-resources/data/xquery/xquery.xml:743]: please use `<literal>` around `exist:serialize`

**Review by duncdrum — CHANGES_REQUESTED** [review]

**Review by duncdrum — APPROVED** [review]
thank you looking good

**Pull request merged**